etc/profile-a-l/luarocks.profile

```diff
@@ -0,0 +1,67 @@
+# Firejail profile for luarocks
+# Description: LuaRocks is the package manager for the Lua programming language.
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include luarocks.local
+# Persistent global definitions
+include globals.local
+
+# disable blacklist for lua interpreter paths
+# Allow lua (blacklisted by disable-interpreters.inc)
+include allow-lua.inc
+
+include disable-common.inc
+# luarocks can invoke compilers
+#include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+# luarocks is hacky and needs shell access
+#include disable-shell.inc
```

Comment:
`include disable-X11.inc`

Reply:
done. Looking at other programs `include disable-X11.inc` could be probably used in all occurences of `blacklist /tmp/.X11-unix` together with a fresh `disable-Wayland.inc` or so.